Q/QGRAF/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QGRAF"
+version = v"3.6.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://anonymous:anonymous@qgraf.tecnico.ulisboa.pt/v3.6/qgraf-$(version).tgz", "648cc82bf3327a4c36d36847d3921fcfa5d7f73c1646241760db030abdc86c29")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+mkdir fmodules
+mkdir -p "${bindir}"
+${FC} -o "${bindir}/qgraf${exeext}" -Os -J fmodules qgraf-*.f08
+
+mkdir -p ${prefix}/share/QGRAF
+mkdir -p ${prefix}/share/licenses/QGRAF
+
+mv array.sty form.sty phi3 qcd qed qedx qgraf.dat sha256sums sum.sty ${prefix}/share/QGRAF/
+head -93 qgraf-3.6.5.f08 > ${prefix}/share/licenses/QGRAF/license
```

Comment:
Now that I see it, I notice the license of this software is very far from being OSI-approved.  There are many restrictions about distribution of the compiled binary, I'm not very comfortable publishing it.

Reply:
Okay, I see. Let me contact the authors for it.
Thank you for your reviews. @giordano 
Should I close this PR until I contact the authors to authorize this distribution?
Or wait for the authors to clearly reply for authorization or non-authorization?